### DISCUSSION TITLE
Rustdesk.TOML file doesn't have the ID

### DISCUSSION THREAD

**Discussion by billgatos** [rustdesk/rustdesk#4177] [Q&A]
Hi all,

I have in my work environment the GLPI and Rustdesk.
Now GLPI using agent 1.5 gets the info from rustdesk ID from the TOML file located at C:\Windows\ServiceProfiles\LocalService\AppData\Roaming\RustDesk\config
but none of the files, there or anywhere else contains the ID inside of the file to be grabbed to GLPI Inventory.

id = ''
enc_id = '0............... some stuff encrypted'
In stable version 1.19 i was able to get the ID on those files.
I need nightly because of security on the clients.
Best regards

**Comment by rustdesk** [comment]
Yes, it is for security.

https://github.com/rustdesk/rustdesk/blob/789200853575f63a336bce72c1a09a215ecfb5cb/src/core_main.rs#L223

**Comment by ksingh7** [comment]
@rustdesk  `--get-id` and `--password` does not work out of the box ? can you guide how to enable that ? , i would happily write a documentation section on that and submit a PR .
